deploy/sre-prometheus/100-cluster-upgrade.PrometheusRule.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-cluster-upgrade-alerts
+    role: alert-rules
+  name: sre-cluster-upgrade-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-cluster-upgrade-alerts
+    rules:
+    - alert: ClusterUpgradingSRE
+      expr: cluster_version{type="updating"} > 0
+      labels:
+        severity: critical
```

```suggestion
        severity: warning
```

Per Slack discussion, to keep context for it, but not to cause unnecessary metal association with a "critical"